src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total;
+
+  if (days > 2 < 7) {
```

Comment:
Weird condition, if you want to have 2 conditions in one if statement then you need to combine them using logical operators like && or ||